src/main/resources/liquibase/crop_changelog/15_4_1.xml

```diff
@@ -0,0 +1,190 @@
+			(6000,1500,0,'Default Seed Store','DSS',0,0,0,0,0,0,NULL),
+			(6001,410,0,'Default Breeding Location','DBL',0,0,0,0,0,0,NULL);
+		</sql>
+	</changeSet>
+
+	<changeSet id="v15.4.1-3" author="rowena">
+		<preConditions onFail="MARK_RAN">
+			<and>
+				<columnExists tableName="location" columnName="ldefault"/>
+				<sqlCheck expectedResult="0">
+					SELECT COUNT(1) FROM location l
+					WHERE l.ldefault = 1 AND l.ltype =
+					(SELECT u.fldno FROM udflds u WHERE u.ftable='LOCATION' AND u.ftype='LTYPE' AND fcode='SSTORE')
+				</sqlCheck>
+				<sqlCheck expectedResult="1">
+					SELECT COUNT(1) FROM location WHERE locid=6000 AND AND ltype = 1500;
+				</sqlCheck>
+			</and>
+		</preConditions>
+		<comment>SET DSS AS DEFAULT STORAGE location</comment>
+		<sql dbms="mysql">
+			UPDATE location SET ldefault = 1 WHERE locid=6000;
+		</sql>
+	</changeSet>
+	
+	<changeSet id="v15.4.1-4" author="rowena">
+		<preConditions onFail="CONTINUE">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(1) FROM udflds u WHERE  fcode = "ANCEST";
+			</sqlCheck>
+		</preConditions>
+		<comment>SET </comment>
+        <sql dbms="mysql">
+			UPDATE udflds SET fcode  = "ANCEST_DATA" WHERE fcode = "ANCEST" AND ftable = "ATRIBUTS" AND fldno>1551;
+		</sql>
+	</changeSet>
+
+	<changeSet id="v15.4.1-5" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*) FROM projectprop WHERE variable_id=8251 AND project_id IN (SELECT project_id FROM project WHERE `name` LIKE '%Nursery Template%');
+			</sqlCheck>
+		</preConditions>
+		<comment>Remove variable STUDY_BM_CODE from nursery template. IBP-3928</comment>
+		<sql dbms="mysql">
+			DELETE FROM projectprop WHERE variable_id=8251 AND project_id IN (SELECT project_id FROM project WHERE `name` LIKE '%Nursery Template%');
+		</sql>
+	</changeSet>
+
+	<changeSet id="v15.4.1-6" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM cvterm where cvterm_id=4021; 
+			</sqlCheck>
+		</preConditions>
+		<comment>Change method of variables with same property-method-scale triplet. IBP-3800</comment>
+		<sql dbms="mysql">
+			SET FOREIGN_KEY_CHECKS=0;
+			
+			REPLACE INTO `cvterm`(`cvterm_id`,`cv_id`,`name`,`definition`,`dbxref_id`,`is_obsolete`,`is_relationshiptype`) VALUES (4021,1020,'Applied at plot level','Applied at plot level',NULL,0,0);
+
+			UPDATE cvterm_relationship SET object_id=4021 WHERE type_id=1210 AND subject_id IN (8258,8259,8253);
+			
+			SET FOREIGN_KEY_CHECKS=1;
+		</sql>
+	</changeSet>
+
+       <changeSet id="v15.4.1-7" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">SELECT count(*) > 1 from cvterm WHERE `name` IN ('EW_DW_kgPlot','EW_FW_gPlot','EW_FW_kgPlot');
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Apply correction on methods of variables EW_DW_kgPlot, EW_FW_gPlot, EW_FW_kgPlot. IBP-3920
```

Comment:
Is IBP-3920 the correct JIRA issue for this changeset?

Reply:
The correct ticket is IBP-3930